pandas/core/common.py

```diff
@@ -1857,23 +1857,84 @@ def _maybe_box_datetimelike(value):
 
 _values_from_object = lib.values_from_object
 
-
-def _possibly_convert_objects(values,
-                              datetime=True,
-                              numeric=True,
-                              timedelta=True,
-                              coerce=False,
+# TODO: Remove in 0.18 or 2017, which ever is sooner
+def _possibly_convert_objects(values, convert_dates=True,
```

Comment:
Not quite -- I only want soft conversion (like the `pd.Series` constructor). I don't want to convert text that matches a regex, only an array of all `datetime` instances (to datetime64) or all `timedelta` instances (to timedelta64).